docs/diracdoctools/Utilities.py

```diff
@@ -3,7 +3,7 @@
 import logging
 import os
 import sys
-import subprocess
+import subprocess32 as subprocess
```

Comment:
As the diracdoctools run outside of a DIRAC(os) installation, can you add subprocess32 explicitly in `docs/requirements.txt` please.